helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -63,10 +56,33 @@ controller:
   ##
   minReadySeconds: 10
 
+
+  ## Node tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  tolerations: {{- toYaml .Values.ingressNginx.controller.tolerations | nindent 4 }}
+
+  ## Affinity and anti-affinity
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {{- toYaml .Values.ingressNginx.controller.affinity | nindent 4 }}
+
   ## terminationGracePeriodSeconds
+  ## wait up to five minutes for the drain of connections
```

Comment:
Isn't this 1 minute?

Reply:
in our case yes :) I copy paste the default text
I will fix it